### ISSUE TITLE
Fungal, Environmental, Microbial Collections in Arctos?

### ISSUE THREAD

**Issue by campmlc** [ArctosDB/arctos#8200]
I am reaching out to see how many institutions have or are planning to have microbial/environmental/fungal collections in Arctos. MSB may be incorporating these kinds of samples, and we'd like to discuss and collaborate with others.
@happiah-madson @falco-rk @dandistel @RJSS8

**Comment by happiah-madson** [comment]
We have frozen isolated bacteria cultures! We also have a handful of environmental samples, but emphasis on handful.

**Comment by dustymc** [comment]
> they arent a part 

Just catalog what you have, Arctos has things and concepts separated for just this reason.

**Comment by campmlc** [comment]
UAM records the as substrate as the taxon - eg water screened for frog DNA. Would the DNA be the part?

**Comment by dustymc** [comment]
>Would the DNA be the part?

Just catalog what you have. 

If you have water (eg because you scooped up a puddle), catalog that. 
If you have filter paper (eg because you poured the water through it), catalog that. 
If you have DNA (eg because you've magicked it out of the filter paper), catalog that.
If you have only the record of someone doing any of that, catalog that.

**Comment by campmlc** [comment]
Yes, including negative observations. We screened this water for x using y method  on z date and found nothing.

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by dustymc** [comment]
>found nothing.

And so you have nothing new and no need to catalog anything else. The experiment should be recorded in a publication (and the publication is a great method for a https://arctos.database.museum/info/ctDocumentation.cfm?table=ctattribute_type#examined_for determination), and if you've cataloged and usefully cited the water I can repeat your experiment and everything's right with the world.

**Comment by camwebb** [comment]
We're just setting off with a fungal collection, UAM:Myco, 450 records to date. Classification imported from [FungalNames](https://nmdc.cn/fungalnames/).
